tests/storage/hotplug.go

```diff
@@ -1746,6 +1747,104 @@ var _ = SIGDescribe("Hotplug", func() {
 		})
 	})
 
+	// Some of the functions used here don't behave well in parallel (CreateSCSIDisk).
+	// The device is created directly on the node and the addition and removal
+	// of the scsi_debug kernel module could create flakiness in parallel.
+	Context("[Serial]Hotplug LUN disk", Serial, func() {
+		var (
+			nodeName, address, device string
+			pvc                       *corev1.PersistentVolumeClaim
+			pv                        *corev1.PersistentVolume
+			vm                        *v1.VirtualMachine
+		)
+
+		BeforeEach(func() {
+			nodeName = tests.NodeNameWithHandler()
+			address, device = tests.CreateSCSIDisk(nodeName, []string{})
+			By(fmt.Sprintf("Create PVC with SCSI disk %s", device))
+			pv, pvc, err = tests.CreatePVandPVCwithSCSIDisk(nodeName, device, util.NamespaceTestDefault, "scsi-disks", "scsipv", "scsipvc")
```

Comment:
Do the namesace always match for `util.NamespaceTestDefault` and the `testsuite.GetTestNamespace`? Could happen that you create the PVC and the VM in 2 different namespaces?

Reply:
I guess it shouldn't be a problem since I'm using nil as `GetTestNamespace` argument, but will only use the latter for consistency.